custom_components/drivvo/sensors.py

```diff
@@ -128,7 +126,6 @@ class DrivvoSensorEntityDescription(SensorEntityDescription):
         name="Refuelling Price",
         icon="mdi:cash",
         device_class=SensorDeviceClass.MONETARY,
-        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Fixes:
```
2025-03-03 00:42:33.058 WARNING (MainThread) [homeassistant.components.sensor] Entity sensor.xxxxxxxxxxxxxx_preco_do_combustivel (<class 'custom_components.drivvo.sensor.DrivvoSensorEntity'>) is using state class 'measurement' which is impossible considering device class ('monetary') it is using; expected None or one of 'total'; Please update your configuration if your entity is manually configured, otherwise create a bug report at https://github.com/hudsonbrendon/sensor.drivvo/issues
```

Reply:
Manter a classe de dispositivo aqui causa warnings nos logs…